### ISSUE TITLE
German date format

### ISSUE THREAD

**Issue by thorstengrothe** [Bruce-Connor/org-agenda-property#3]
I customized a custom agenda command for my german exams look here

![grafik](https://cloud.githubusercontent.com/assets/19598706/23887582/1e4a61e4-0883-11e7-8430-6f3910fc4e05.png)

I would like to have a more german like format for the dates on the right side like this

[Di 28.03.2017]

for orgmode I customized like this

` (setq org-time-stamp-custom-formats
        '("<%a %d.%m.%Y>" . "<%a %d.%m.%Y %H:%M>"))`

but this does not work in my custom agenda command.

Do you have an idea?

Regards 
Thorsten

**Comment by Malabarba** [comment]
No, I'm afraid I don't know how to control that. But if you figure out feel free to open a PR.
